tests/pkcli/fmt_test.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""test fmt
+
+:copyright: Copyright (c) 2022 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from multiprocessing.dummy import active_children
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdp
+import pytest
+
+
+def test_run():
+    from pykern import pkunit
+    from pykern import pkio
+    from pykern.pkcli import fmt
+
+    for d in pkunit.case_dirs():
+        if d == '/home/vagrant/src/radiasoft/pykern/tests/pkcli/fmt_work/single_file':
```

Comment:
No hardwired paths and I don't think this is necessary. Just fmt.run on the directory.